src/collector.ts

```diff
@@ -132,6 +157,18 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        function checkReplaceDependency(dep: IDependency): any {
+            let replaceDependency = replaceDependencies.get(dep.name.value + "@" + dep.version.value);
+            if (replaceDependency === undefined) {
+                replaceDependency = replaceDependencies.get(dep.name.value);
+                if(replaceDependency === undefined) {
+                    //dependencies.push(dep);
+                    return null;
```

Comment:
Once small improvement which will simply caller code. 
Instead of returning 'null', return same deps in case no replacement is found. So that at caller place, you don't have to check and handle null condition.